packages/terra-form-select/tests/wdio/select-spec.js

```diff
@@ -2376,6 +2376,17 @@ Terra.describeViewports('Select', ['tiny'], () => {
       it('should display selected option', () => {
         Terra.validates.element('tag controlled selected option');
       });
+
+      it('should focus on pressing tab key', () => {
```

Comment:
Does this test validates disabled tag navigations as well?

Reply:
@supreethmr I have added tests to validate disabled tag navigations